source/includes/api/_widget.md

```diff
@@ -1,428 +1,296 @@
 
+> Return all widgets associated with two datasets, that are associated with either `rw` or `prep` applications 
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget/51851e22-1eda-4bf5-bbcc-cde3f9a3a943?includes=metadata
-```
-
-> Example response:
-
-```json
-{
-    "data": {
-        "id": "51851e22-1eda-4bf5-bbcc-cde3f9a3a943",
-        "type": "widget",
-        "attributes": {
-            "name": "Example Carto Dataset6",
-            "dataset": "be76f130-ed4e-4972-827d-aef8e0dc6b18",
-            "slug": "example-carto-dataset6",
-            "userId": "5820ad9469a0287982f4cd18",
-            "description": null,
-            "source": null,
-            "sourceUrl": null,
-            "authors": null,
-            "application": [
-                "rw"
-            ],
-            "verified": false,
-            "default": false,
-            "protected": false,
-            "defaultEditableWidget": false,
-            "published": true,
-            "freeze": false,
-            "env": "production",
-            "queryUrl": null,
-            "widgetConfig": "{}",
-            "template": false,
-            "layerId": null,
-            "createdAt": "2017-02-08T15:30:34.505Z",
-            "updatedAt": "2017-02-08T15:30:34.505Z",
-            "metadata": [
-              {
-                "id": "5aeb1c74a096b50010f3843f",
-                "type": "metadata",
-                "attributes": {
-                  "dataset": "86777822-d995-49cd-b9c3-d4ea4f82c0a3",
-                  "application": "rw",
-                  "resource": {
-                    "id": "51851e22-1eda-4bf5-bbcc-cde3f9a3a943",
-                    "type": "widget"
-                  },
-                  "language": "en",
-                  "info": {
-                    "caption": "t",
-                    "widgetLinks": []
-                  },
-                  "createdAt": "2018-05-03T14:28:04.482Z",
-                  "updatedAt": "2018-06-07T11:30:40.054Z",
-                  "status": "published"
-                }
-              }
-            ]
-          }
-        }
-    }
-}
+curl -X POST "https://api.resourcewatch.org/widget/find-by-ids" \
+-H "Authorization: Bearer <your-token>" \
+-H 'Content-Type: application/json' \
+-d '{
+	"ids": ["<dataset 1 id>", "<dataset 2 id>"],
+    "app" "rw,prep"
+}'
 ```
 
-#### Requesting multiple additional entities
-
-You can request multiple related entities in a single request using commas to separate multiple keywords
+> Return all widgets associated with two datasets, that are associated with both `rw` and `prep` applications simultaneously 
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget/51851e22-1eda-4bf5-bbcc-cde3f9a3a943?includes=metadata,user,vocabulary
+curl -X POST "https://api.resourcewatch.org/widget/find-by-ids" \
+-H "Authorization: Bearer <your-token>" \
+-H 'Content-Type: application/json' \
+-d '{
+	"ids": ["<dataset 1 id>", "<dataset 2 id>"],
+    "app" "rw@prep"
```

Comment:
This filter by application syntax looks very similar to the layer endpoint...is it worth factoring out into a more general section, or is it just in these two places?